registry/time/helpers_test.go

```diff
@@ -0,0 +1,58 @@
+package time
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
```

Comment:
You asked me to make computeTimeFromFormat unexported, but then I have to switch the test into time package, and not the time_test one, so I'm adding testify

maybe it's what you were talking here https://github.com/go-sprout/sprout/pull/94/files#r1870643709

What would you suggest ?

Reply:
`helpers_test.go` is always in the same package, while `functions_test.go` uses `package_test` to cover this use case, so everything is good here ✅